cypress/e2e/pages/onDemandAudio/index.cy.js

```diff
@@ -1,32 +1,487 @@
+  {
+    path: '/arabic/bbc_arabic_radio/programmes/p030vh2y', // On Demand Brand
+    service: 'arabic',
+    runforEnv: 'live',
+    tests,
+  },
+  {
+    path: '/arabic/bbc_arabic_radio/programmes/p030vh25', // On Demand Brand
+    service: 'arabic',
+    runforEnv: 'test',
+    tests,
+  },
+  {
+    path: '/arabic/bbc_arabic_radio/w172y3wn75cm441', // On Demand Episode
+    service: 'arabic',
+    runforEnv: ['test', 'live'],
+    tests,
+  },
+  {
+    path: '/arabic/podcasts/p02pc9qc', // Podcast Brand
+    service: 'arabic',
+    runforEnv: ['local', 'test', 'live'],
+    tests,
+  },
+  {
+    path: '/arabic/podcasts/p02pc9qc/p09kwwfs', // Podcast Episode
+    service: 'arabic',
+    runforEnv: ['test', 'live'],
+    tests,
+  },
+  {
+    path: '/bengali/bbc_bangla_radio/programmes/p030vjwg', // On Demand Brand
+    service: 'bengali',
+    runforEnv: ['live'],
+    tests,
+  },
+  {
+    path: '/bengali/bbc_bangla_radio/programmes/p030vjwm', // On Demand Brand
+    service: 'bengali',
+    runforEnv: ['live'],
+    tests,
+  },
+  {
+    path: '/bengali/bbc_bangla_radio/w172xwdq2b07ywv', // On Demand Episode
+    service: 'bengali',
+    runforEnv: ['test', 'live'],
+    tests,
+  },
+  {
+    path: '/burmese/bbc_burmese_radio/programmes/p0340rnm', // On Demand Brand
+    service: 'burmese',
+    runforEnv: ['test', 'live'],
+    tests,
+  },
+  {
+    path: '/burmese/bbc_burmese_radio/w3ct1m6n', // On Demand Episode
+    service: 'burmese',
+    runforEnv: ['test', 'live'],
+    tests,
+  },
+  {
+    path: '/burmese/podcasts/p02pc9lh', // Podcast brand
+    service: 'burmese',
+    runforEnv: ['test', 'live'],
+    tests,
+  },
+  {
+    path: '/burmese/podcasts/p02pc9lh/p09kzply', // Podcast Episode
+    service: 'burmese',
+    runforEnv: ['test', 'live'],
+    tests,
+  },
+  {
+    path: '/gahuza/bbc_gahuza_radio/programmes/p0340x2m', // On Demand Brand
+    service: 'gahuza',
+    runforEnv: ['test', 'live'],
+    tests,
+  },
+  {
+    path: '/gahuza/bbc_gahuza_radio/w3ct1v5v', // On Demand Episode
+    service: 'gahuza',
+    runforEnv: ['test', 'live'],
+    tests,
+  },
+  {
+    path: '/gahuza/podcasts/p07yh8hb', // Podcast Brand
+    service: 'gahuza',
+    runforEnv: ['local', 'test', 'live'],
+    tests,
+  },
+  {
+    path: '/gahuza/podcasts/p07yh8hb/p09km4t4', // Podcast Episode
+    service: 'gahuza',
+    runforEnv: ['test', 'live'],
+    tests,
+  },
+  {
+    path: '/hausa/bbc_hausa_radio/programmes/p030s4mh', // On Demand Brand
+    service: 'hausa',
+    runforEnv: ['live'],
+    tests,
+  },
+  {
+    path: '/hausa/bbc_hausa_radio/programmes/p030s4mx', // On Demand Brand
+    service: 'hausa',
+    runforEnv: ['test'],
+    tests,
+  },
+  {
+    path: '/hausa/bbc_hausa_radio/w3ct1qzn', // On Demand Episode
+    service: 'hausa',
+    runforEnv: ['live'],
+    tests,
+  },
+  {
+    path: '/hausa/bbc_hausa_radio/w3ct1033', // On Demand Episode
+    service: 'hausa',
+    runforEnv: ['test'],
+    tests,
+  },
```

Comment:
Because test points to live data, we can pick one path and combine them into a single entry, with runforEnv: ['test', 'live']